models/GATStack.py

```diff
@@ -16,8 +16,8 @@ def __init__(
         num_nodes: int,
         hidden_dim: int,
         config_heads: {},
-        heads: int = 1,
-        negative_slope: float = 0.2,
+        heads: int = 6,
```

Comment:
@pzhanggit 
why are you setting as default a number of heads equal to 6? Shouldn't it remain 1?

Reply:
> @pzhanggit
> why are you setting as default a number of heads equal to 6? Shouldn't it remain 1?

To be more expressive. Also, 6 is the number found to allow the model to pass the unit test.